tests/test_config_parser.py

```diff
@@ -812,7 +812,7 @@ def prepare_mkosi_conf_d_2(self, directory: str) -> None:
         ini_lines = [
             "[Content]",
             "Packages=[ vim,!vi",
-            "  ca-certificates, bzip ]" "",
+            "  ca-certificates, bzip ]  ",
```

Comment:
Hmm, this adds a trailing whitespace, no?

Reply:
Yes, indeed! I forgot I wanted to put that into a separate commit. Before, this did an implicit string concatenation, adding an empty string to the line, so a noop that's already done when compiling the byte code. I though this would be more realistic as a test case if the line had some trailing whitespace and the tests pass either way. This should have been a separate commit, though, explaining the reasoning, sorry.